content/applications/inventory_and_mrp/inventory/product_management/inventory_valuation/valuation_by_lots.rst

```diff
@@ -0,0 +1,212 @@
+
+To view the valuation layer, go to :menuselection:`Inventory app --> Reporting --> Valuation`. The
+:guilabel:`Total Value` per unit matches the cost on the product form.
+
+.. example::
+   Continuing the example in the table above, when the product cost is `$15`, the valuation for a
+   newly created `LOT3` is also be `$15`.
+
+   .. image:: valuation_by_lots/create-new.png
+      :align: center
+      :alt: Show inventory adjustment valuation.
+
+.. _inventory/product_management/valuation-cost-existing:
+
+Existing lot/serial number
+--------------------------
+
+When adjusting the quantity of an existing lot/serial number, the value is based on the most recent
+valuation layer that specific lot/serial number.
+
+.. example::
+   Continuing the example in the table above, LOT 1's value is $10.
+
+   So, when the quantity is updated from `1.00` to `2.00`, the additional quantity is also valued at
+   `$10`, reflecting the latest valuation layer for `LOT 1`.
+
+   .. figure:: valuation_by_lots/existing.png
+      :align: center
+      :alt: Show valuation of LOT 1 getting updated.
+
+      The inventory adjustment (top line) is valued the same as LOT 1 (bottom line).
+
+.. _inventory/product_management/view-valuation:
+
+View valuation
+==============
+
+To find the average cost of a specific lot/serial number, go to :menuselection:`Inventory app -->
+Products --> Lots/Serial Numbers` and select the desired record.
+
+Both the :guilabel:`Cost` and :guilabel:`Average Cost` fields show a unit's average cost. The
+:guilabel:`Total Value` reflects the total on-hand value for that lot/serial number.
+
+.. figure:: valuation_by_lots/lot.png
+   :align: center
+   :alt: Show cost of the lot/serial number.
+
+   Lot form, displaying **Cost** field. The **Valuation** smart button is in the top-right.
+
+Valuation layers of a lot/serial number can be viewed through the :ref:`valuation report
+<inventory/product_management/valuation-report>` or by clicking the lot/serial number's
+:guilabel:`Valuation` smart button. These detailed, line-by-line records can help determine how each
+inventory move of the specific lot/serial number affects its valuation.
+
+.. _inventory/product_management/valuation-report:
+
+Valuation report
+----------------
+
+Display the valuation of lots and serial numbers in the database by going to
+:menuselection:`Inventory app --> Reporting --> Valuation`.
+
+On the resulting :guilabel:`Stock Valuation` report, click the search bar, and in the
+:icon:`oi-group` :guilabel:`Group By` section of the resulting drop-down menu, select
+:guilabel:`Lot/Serial number`.
+
+.. tip::
+   Click the :icon:`fa-plus` :guilabel:`(plus)` icon to the right of a collapsed lot number line to
+   :ref:`manually modify the cost <inventory/warehouses_storage/update-unit-price>`.
+
+   This is useful for adjusting individual lot prices when a purchase order or bill includes
+   multiple lots/serial numbers, as initial prices are identical upon reception.
```

Comment:
I am not seeing any 'plus' icon to the right of a collapsed lot number line -- at least, not in the Master runbot I am using for this review. Please double-check this during your revisions to ensure the accuracy of this statement.

Reply:
You'll have to make sure the costing method is set to avco or fifo, and process a receipt, to see the plus button (: